addons/account/views/res_config_settings_views.xml

```diff
@@ -372,11 +372,6 @@
                                 <field name="quick_edit_mode" placeholder="Disabled"/>
                             </setting>
                         </block>
-                        <block title="Audit Trail" name="audit_trail_setting_container">
```

Comment:
I know it's move in `account_accountant`, but it should stay a setting in community IMO

Reply:
Because it's a community feature.

I don't understand why it would be in the reporting section tbh